streams/src/main/java/org/apache/kafka/streams/processor/internals/StreamThread.java

```diff
@@ -496,9 +496,7 @@ private void runLoop() {
         while (isRunning() || taskManager.isRebalanceInProgress()) {
             try {
                 runOnce();
-                if (assignmentErrorCode.get() == AssignorError.VERSION_PROBING.code()) {
-                    log.info("Version probing detected. Rejoining the consumer group to trigger a new rebalance.");
-
+                if (assignmentErrorCode.get() == AssignorError.REBALANCE_NEEDED.code()) {
```

Comment:
Are we deprecating the VERSION_PROBING?

Reply:
This isn't a client or externally-facing config, just something each thread sets internally if it detects version probing during `onAssignment`
If you're asking about the old log message, I moved it to where we set the error code.